reana_commons/config.py

```diff
@@ -130,3 +130,6 @@
 
 REANA_STORAGE_BACKEND = os.getenv('REANA_STORAGE_BACKEND', 'LOCAL')
 """Storage backend deployed in current REANA cluster."""
+
+REANA_WORKFLOW_UMASK = 0o02
```

Comment:
It would be clearer tot use `0o002` (note three digits) or even `0o0002` (by analogy with `0o0644` and friends).

The new `pytest-reana==0.5.0.dev20190321` is released on PyPI, so you can upgrade the dependency version also.